src/GetDocumentInsider/Commands/GetDocumentCommandWorker.cs

```diff
@@ -109,7 +109,7 @@ public static bool TryProcess(GetDocumentCommandContext context, IServiceProvide
                         return false;
                     }
 
-                    var finished = Task.WhenAny(resultTask, Task.Delay(TimeSpan.FromMinutes(1)));
+                    var finished = Task.WhenAny(resultTask, Task.Delay(TimeSpan.FromMinutes(1))).Result;
```

Comment:
No, I don't know how this ever worked. I may have been testing an old copy of the package at one point 😦